pkg/test/e2eutil/wait_util.go

```diff
@@ -29,6 +31,20 @@ import (
 // This can be used in multiple ways, like verifying that a required resource is ready before trying to use it, or to test
 // failure handling, like simulated in SimulatePodFail.
 func WaitForDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace, name string, replicas int, retryInterval, timeout time.Duration) error {
+	return waitForDeployment(t, kubeclient, namespace, name, replicas, retryInterval, timeout, false)
+}
+
+// WaitForOperatorDeployment has the same functionality as WaitForDeployment but will no wait for the deployment if the
+// test was run with a locally run operator (--up-local flag)
+func WaitForOperatorDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace, name string, replicas int, retryInterval, timeout time.Duration) error {
+	return waitForDeployment(t, kubeclient, namespace, name, replicas, retryInterval, timeout, true)
+}
+
+func waitForDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace, name string, replicas int, retryInterval, timeout time.Duration, isOperator bool) error {
+	if isOperator && test.Global.LocalOperator {
+		t.Logf("Operator is running locally; skip waitForDeployment\n")
```

Comment:
```suggestion
		t.Log("Operator is running locally; skip waitForDeployment\n")
```

Reply:
Shouldn't the `\n` be left off when using `Log` vs `Logf`?

Reply:
You're right. Updated the suggestion.